### PR TITLE
Update italian.c

### PR DESCRIPTION
**Pull request by berseker**
Please check this updated Italian translation

### CODE REVIEW

File: src/translation/italian.c [hunk 1]
```diff
@@ -812,6 +865,90 @@ static translation_string all_strings[] = {
...
+    {TR_BUILDING_BRICKWORKS_MANY_EMPLOYEES, "Questa fabbrica sta lavorando meno della capacità massima. La produzione di mattoni potrebbe essere più efficiente con più lavoratori."},
+    {TR_BUILDING_BRICKWORKS_HALF_EMPLOYEES, "Questa fabbrica è a corto di personale. Impiega più tempo del dovuto per produrre mattoni."},
+    {TR_BUILDING_BRICKWORKS_SOME_EMPLOYEES, "Qui lavorano pochi lavoratori. La produzione di mattoni di conseguenza è lenta."},
+    {TR_BUILDING_BRICKWORKS_FEW_EMPLOYEES, "Non avendo praticamente nessun lavoratore, la produzione è sostanzialmente ferma. Qui verranno prodotti pochissimi mattoni nel corso dell'anno."},
+    {TR_BUILDING_BRICKWORKS_NO_RESOURCES, "Questa fabbrica necessita di forniture di argilla e sabbia, da un magazzino o una cava di argilla e di sabbia, per produrre mattoni."},
+    {TR_BUILDING_BRICKWORKS_STORED_SAND, "Sabbia stoccata,"},
+    {TR_BUILDING_BRICKWORKS_STORED_CLAY, "Argilla stoccata,"},
+    {TR_BUILDING_CONCRETE_MAKER, "Fabbrica di cemento"},
+    {TR_BUILDING_CONCRETE_MAKER_DESC, "Qui, la sabbia è mischiata per produrre cemento per la costruzione di monumenti."},
+	{TR_BUILDING_CONCRETE_MAKER_PRODUCTION, "La produzione è "},
```
Comment (luigimannoni) [inline src/translation/italian.c:918]: Tab and whitespace at the end of string

File: src/translation/italian.c [hunk 2]
```diff
@@ -849,15 +986,366 @@ static translation_string all_strings[] = {
...
+    {TR_PARAMETER_DISPLAY_APPLY_COST, "applica costo"},
+    {TR_PARAMETER_DISPLAY_NO_COST, "nessun costo"},
+    {TR_PARAMETER_DISPLAY_ADD_AS_BUYING, "Aggiungi come compratore"},
+    {TR_PARAMETER_DISPLAY_ADD_AS_SELLING, "Aggiungi come venditore"},
+    {TR_PARAMETER_DISPLAY_DESTROY_ALL_TYPES, "distruggi tutti i tipi"},
+    {TR_ORDER_CONDITION_NEVER, "Mai" },
+    {TR_ORDER_CONDITION_ALWAYS, "Sempre" },
+    {TR_ORDER_CONDITION_SOURCE_HAS_MORE_THAN, "L'origine ha più di" },
+    {TR_ORDER_CONDITION_DESTINATION_HAS_LESS_THAN, "La destinazione ha meno di" },
+    {TR_BUILDING_INFO_DEPOT_SELECT_SOURCE, "Scegli una fonte"},   {TR_BUILDING_INFO_DEPOT_SELECT_DESTINATION, "Scegli una destinazione"},
```
Comment (luigimannoni) [inline src/translation/italian.c:1303]: 2 rows in the same line

**Review by luigimannoni — CHANGES_REQUESTED** [review]
Some changes needed mostly for whitespace and wrong round parenthesis `(` instead of braces `{`.

Translation is 100% accurate, however there are few rows to change:

- Fetching/not fetching: should have an italian translation, not sure where the string appears so my best guess should be either "Recuperando" or "Prendendo".
- "Denarii"/"Denari": New translation rows have "Denarii" some old rows have "Denari", I'd like to see "Denarii" rather than the italianization with a single plural "i". Also [wikipedia](https://it.wikipedia.org/wiki/Denario) refers it with 2 ii in the plural form which I think was the proper roman name.

**Comment by berseker** [comment]
just committed a revision - thank you for the comments!

**Review by luigimannoni — APPROVED** [review]

**Comment by berseker** [comment]
i do not know how to fix in order to merge this to master

**Comment by Ouaz** [comment]
End of line 515,  there's a bracket `)` instead of a curly bracket `}`

```
src/translation/italian.c:515:70: error: expected ‘}’ before ‘)’ token
  515 |     {TR_WINDOW_ADVISOR_ENTERTAINMENT_TAVERN_COVERAGE_PLURAL,"Taverne"),
```
Add a commit to fix it, it should be good.

**Comment by berseker** [comment]
> * End of line 515,  there's a bracket `)` instead of a curly bracket `}`
> 
> ```
> src/translation/italian.c:515:70: error: expected ‘}’ before ‘)’ token
>   515 |     {TR_WINDOW_ADVISOR_ENTERTAINMENT_TAVERN_COVERAGE_PLURAL,"Taverne"),
> ```
> 
> Add a commit to fix it, it should be good.
> 
> Ah, no, wait, there's more...
> 
> * Line 648: there's a break line here. The entry must be on a single line.
> 
> `{TR_BUILDING_LARGE_MAUSOLEUM_CONSTRUCTION_DESC, "Per costruire questo mausoleo` `_______ richiede materiale conservato in un magazzino, manovalanza da un campo di lavoro e architetti da una corporazione di architetti." }, `
> 
> * Line 1014: the closing `"` is missing after `azioni`
> 
> ```
> src\translation\italian.c:1014:41: error: missing terminating " character
>  1014 |     {TR_EDITOR_SCENARIO_EVENTS_ACTIONS, "azioni },
> ```
> 
> * Line 1333: the opening `"` is missing before `Porta`
> 
> `{TR_BUILDING_PANELLED_GARDEN_WALL_GATE, Porta delle mura del giardino a pannelli" },`

thank you - stupid errors which I could not see since I still do not have confidence with github editor  (is there a debug function for syntax errors?

I fixed the above points

File: src/translation/italian.c [hunk 3]
```diff
@@ -812,6 +864,90 @@ static translation_string all_strings[] = {
     {TR_BUILDING_GLADIATOR_STATUE, "Statua del gladiatore"},
     {TR_BUILDING_HIGHWAY, "Strada veloce"},
     {TR_BUILDING_HIGHWAY_DESC, "Le strade veloci permettono ai cittadini di raggiungere la destinazione più velocemente."},
+    {TR_BUILDING_GOLD_MINE, "Miniera di oro"},
+    {TR_BUILDING_GOLD_MINE_DESC, "Estrai oro per commerciarlo, o per rifornire la zecca cittadina e produrre denarii."},
+    {TR_BUILDING_GOLD_MINE_PRODUCTION, "La produzione è"},
+    {TR_BUILDING_GOLD_MINE_COMPLETE, "completa."},
+    {TR_BUILDING_GOLD_MINE_HALTED, "
+	Il tuo consigliere del commercio ha ordinato di interrompere la produzione di oro."},
```
Comment (Keriew) [inline src/translation/italian.c:872]: This line should be merged with the previous one.

**Review by Keriew — DISMISSED** [review]
All right, looks like there's one tiny change left and then it's ready to merge.

File: src/translation/italian.c (hunk 3 above)
Comment (crudelios) [inline src/translation/italian.c:872]:
```suggestion
    {TR_BUILDING_GOLD_MINE_HALTED, "Il tuo consigliere del commercio ha ordinato di interrompere la produzione di oro."},
```

**Pull request merged**